lib/sharedFunctions.bash

```diff
@@ -203,7 +203,7 @@ function trackAndTracePostFromFile() {
 	_curlResponse=$(curl -f -s -H 'Content-Type: application/json' -X POST -d "{\"username\":\"${USERNAME}\", \"password\":\"${PASSWORD}\"}" "https://${MOLGENISSERVER}/api/v1/login") \
 		|| {
 			log4Bash 'ERROR' "${LINENO}" "${FUNCNAME:-main}" '0' "Failed to login at ${MOLGENISSERVER}."
-			return
+			return 1
```

Comment:
Weet je zeker dat je `return 1` wil? Dat zorgt er i.c.m. `set -e` voor dat het notificatie script er compleet mee ophoudt als één enkele notificatie niet lukt tenzij je altijd als je deze functies aanroept dat in een if statement doet.